main.md

```diff
@@ -379,38 +379,32 @@ are included for formatting and readability.
 !---
 Figure: Example JWT content of a `DPoP` proof {#dpop-proof}
 
-Of the HTTP content in the request, only the HTTP method and URI are
-included in the DPoP JWT, and therefore only these 2 headers of the request
-are covered by the DPoP proof and its signature.
-The idea is sign just enough of the HTTP data to
-provide reasonable proof-of-possession with respect to the HTTP request. But 
-that it be a minimal subset of the HTTP data so as to avoid the substantial 
-difficulties inherent in attempting to normalize HTTP messages. 
-Nonetheless, DPoP proofs can be extended to contain other information of the
-HTTP request (see also (#request_integrity)).
+DPOP JWT conveys only the HTTP method and the request target.
+This is to provide a reasonable correlation with the HTTP request. 
+Nonetheless, DPoP proofs can be extended to contain other information
+related to the HTTP request (see also (#request_integrity)).
 
 
 
 ## Checking DPoP Proofs {#checking}
 
-To check if a string that was received as part of an HTTP Request is a
-valid DPoP proof, the receiving server MUST ensure that
+To validate a DPoP proof, the receiving server MUST ensure that
 
- 1. that there is not more than one `DPoP` header in the request,
- 1. the string value of the header field is a well-formed JWT,
+ 1. that there is not more than one `DPoP` HTTP request header field,
+ 1. the field value is a well-formed JWT,
  1. all required claims per (#DPoP-Proof-Syntax) are contained in the JWT,
- 1. the `typ` field in the header has the value `dpop+jwt`,
- 1. the algorithm in the header of the JWT indicates an asymmetric digital
+ 1. the `typ` JOSE header parameter has the value `dpop+jwt`,
+ 1. the `alg` JOSE header parameter indicates an asymmetric digital
     signature algorithm, is not `none`, is supported by the
     application, and is deemed secure,
  1. the JWT signature verifies with the public key contained in the `jwk`
-    header of the JWT,
- 1. the `jwk` header of the JWT does not contain a private key,
- 1. the `htm` claim matches the HTTP method value of the HTTP
-    request in which the JWT was received,
- 1. the `htu` claim matches the HTTPS URI value for the HTTP
-    request in which the JWT was received, ignoring any query and
-    fragment parts,
+    JOSE header parameter,
+ 1. the `jwk` JOSE header parameter does not contain a private key,
+ 1. the `htm` claim matches the HTTP method of the
+    current request,
+ 1. the `htu` claim matches the current request target,
+    ignoring any query and
```

Comment:
I don't like that the explanatory text "HTTPS URI value for the HTTP request in which the JWT was received" is lost in the rewrite.  The replacement text "current request target" could easily be misunderstood by readers.

Reply:
Ok, see https://github.com/danielfett/draft-dpop/pull/146/files#r855935610